docs/Validation.md

```diff
@@ -370,6 +370,7 @@ Server-side validation is supported out of the box for `pessimistic` mode only.
 ```
 {
     body: {
+        message: 'Error: my custom message in notification popover',
```

Comment:
I'm confused. Shouldn't the message be outside the body? :thinking: 
See https://github.com/marmelab/react-admin/blob/master/packages/ra-core/src/controller/edit/useEditController.ts#L189 for ref